packages/evolution-frontend/src/actions/Auth.ts

```diff
@@ -44,3 +45,82 @@ export const startDirectTokenLogin = (history: History, location: Location, call
         }
     };
 };
+
+/*
```

Comment:
Don't keep copyright headers in the middle of the file

Reply:
removed